doc/source/whatsnew/v1.0.1.rst

```diff
@@ -74,3 +74,5 @@ Bug fixes
 
 Contributors
 ~~~~~~~~~~~~
+
+.. contributors:: v0.25.3..v1.0.1|HEAD
```

Comment:
this is nice!

Is it needed to remove once it is released? (because you say below "for development versions (before a tag is available)")

Reply:
It's not needed after we tag. Assuming there aren't any bugs, we can leave it like this and it should just use the v1.0.1 tag after it's present.

Reply:
Actually, this should have been `v1.0.0..v1.0.1` instead of `v0.25.3..v1.0.1` I suppose?

Also should this be "HEAD of 1.0.x branch" instead of "HEAD" ?

Reply:
https://github.com/pandas-dev/pandas/pull/31704